cmd/controller/state/imagescan/scanner.go

```diff
@@ -402,6 +402,12 @@ func scanJobSpec(
 	vol volumesAndMounts,
 	tolerations []corev1.Toleration,
 ) *batchv1.Job {
+
+	podLabels := map[string]string{}
+	if cfg.CloudProvider == "aks" {
+		podLabels["azure.workload.identity/use"] = "true"
```

Comment:
Can this cause issues if we have an AKS cluster, but workload identity is not setup?

Reply:
I don't think so, but I have an AKS cluster that I can try it out on.

Reply:
Tested the changes on Azure with workload ID disabled, but labels present and it works as expected.

Reply:
Awesome, thanks 👌